{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "d2c44ea0_9e42f869",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 11,
      "author": {
        "id": 1016895
      },
      "writtenOn": "2021-04-30T11:47:20Z",
      "side": 1,
      "message": "The reason to make the change is not clear. It looks like we\u0027re updating the tool, but adding a bunch of exceptions. This makes it seem like it\u0027s not an improvement. It would be good to mention if follow-on patches are coming to address the exceptions, and to action those follow-on patches to show how we get back to a small, limited number of exceptions which are justified.",
      "revId": "0788b705d0ac6708914d9659ead1a2708c89fdb2",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "6ee7e5c8_fdd8ac42",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1016110
      },
      "writtenOn": "2021-04-30T08:45:26Z",
      "side": 1,
      "message": "Out of curiosity: was 4.2.0 suddenly broken for us? What did it change? (that\u0027s the part which is unclear, the reason of the change).",
      "revId": "0788b705d0ac6708914d9659ead1a2708c89fdb2",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "7145deee_8b94a758",
        "filename": "tox.ini",
        "patchSetId": 2
      },
      "lineNbr": 124,
      "author": {
        "id": 1011165
      },
      "writtenOn": "2021-04-29T18:33:17Z",
      "side": 1,
      "message": "See Jobs results from patchset #1",
      "revId": "0788b705d0ac6708914d9659ead1a2708c89fdb2",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}